src/harness/sas.py

```diff
@@ -332,3 +342,113 @@ def _GetDefaultAdminSSLCertPath(self):
   def _GetDefaultAdminSSLKeyPath(self):
     return os.path.join('certs', 'admin_client.key')
 
+
+class HttpServer(httpserver_interface.HttpServerInterface):
```

Comment:
Is there any reason not to derive from the HTTPServer class directly? That might reduce confusion.